api/src/main/resources/moduleApplicationContext.xml

```diff
@@ -132,6 +132,11 @@
                 <property name="appointmentDao">
                     <ref bean="appointmentDao"/>
                 </property>
+                <property name="appointmentConflictTypes">
+                    <list>
+                        <bean class="org.openmrs.module.appointments.conflicts.impl.AppointmentServiceUnavailabilityConflict"/>

```

Comment:
The intention looks like expecting a list, probably you need to give interface in bean class?